client/app/dedicatedCloud/datacenter/drp/steps/common/first/dedicatedCloud-datacenter-drp-firstStep.html

```diff
@@ -0,0 +1,59 @@
+<div class="text-center"
```

Comment:
Why don't you have a unique loader for all steps ?

Reply:
Loading of some steps can be handle by `$transitions`, other not, let's postpone this in the next feature step